tools/c7n_azure/c7n_azure/resources/vm.py

```diff
@@ -23,4 +24,21 @@ class resource_type(object):
         client = 'ComputeManagementClient'
         enum_spec = ('virtual_machines', 'list_all')
         id = 'id'
-        name = 'name'
\ No newline at end of file
+        name = 'name'
+        default_report_fields = (
+            'name',
+            'location',
+            'properties.hardwareProfile.vmSize',
+        )
+
+@VirtualMachine.filter_registry.register('instance-view')
+class InstanceViewFilter(ValueFilter):
+    schema = type_schema('instance-view', rinherit=ValueFilter.schema)
+
+    def __call__(self, i):
```

Comment:
We should keep the practice of having an example of the policy in comments

Reply:
Ah good catch, I'll add unit tests and docs on this too if we go ahead with it.